installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -1,28 +1,37 @@
-[id='proc-install-rhdh-ocp-operator_{context}']
-
+[id="proc-install-rhdh-ocp-operator_{context}"]
 = Deploying {product} on {ocp-short} using the Operator
 
-As a developer, you can install {product} on your OpenShift Container Platform using the {product} Operator.
+As a developer, you can deploy a {product} instance on {ocp-short} by using the *Developer* perspective in the web console.
 
 .Prerequisites
-* Your administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
-* You have created a project in {ocp-short} for your {product} instance. For more information about creating a project in OpenShift, see link:https://docs.openshift.com/container-platform/4.14/applications/projects/working-with-projects.html#odc-creating-projects-using-developer-perspective_projects[Red Hat OpenShift documentation].
+
+* A cluster administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
 
 .Procedure
-. From the *Developer* perspective on the {product-short} web console, click *+Add*.
+
+. Create a project in {ocp-short} for your {product} instance, or select an existing project.
++
+[TIP]
+====
+For more information about creating a project in {ocp-short}, see link:https://docs.openshift.com/container-platform/4.15/applications/projects/working-with-projects.html#creating-a-project-using-the-web-console_projects[Creating a project by using the web console] in the {ocp-brand-name} documentation.
+====
+. From the *Developer* perspective on the {ocp-short} web console, click *+Add*.
 . From the *Developer Catalog* panel, click *Operator Backed*.
 . In the *Filter by keyword* box, enter _{product-short}_ and click the *{product}* card.
 . Click *Create*.
-. Optionally, configure the {product-short} instance with non-default settings.
-. From the *{product}* page, click *Create*.
-. From the *Create Backstage* page, click *Create*
-. From the *Topology* tab, click the *Open URL* icon on the {product-short} pod to start using the {product} platform.
+. Optionally, configure the {product-short} instance with custom settings.
+. On the *Create Backstage* page, click *Create*
```

Comment:
I am wondering if we could squeeze the validation steps with something like:

... click the *{product}* card, and validate the confirmation screens.

Reply:
I don't think it makes sense to do that, since clicking Create is still a part of the procedure, not the validation.